maintenance/cleanupTitles.php

```diff
@@ -76,7 +76,7 @@ protected function fileExists( $name ) {
 	protected function moveIllegalPage( $row ) {
 		$legal = 'A-Za-z0-9_/\\\\-';
 		$legalized = preg_replace_callback( "!([^$legal])!",
-			array( &$this, 'hexChar' ),
+			array( $this, 'hexChar' ),
 			$row->page_title );
 		if ( $legalized == '.' ) $legalized = '(dot)';
 		if ( $legalized == '_' ) $legalized = '(space)';
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add curly braces around the nested statement(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') 2 statements were found on this line. Reformat the code to have only one statement per line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS122)